src/libraries/Common/src/Microsoft/Win32/SafeHandles/SafeEvpKdfHandle.Unix.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    internal sealed class SafeEvpKdfHandle : SafeHandle
+    {
+        public SafeEvpKdfHandle() : base(0, ownsHandle: true)
+        {
+        }
+
+        protected override bool ReleaseHandle()
+        {
+            Interop.Crypto.EvpKdfFree(handle);
+            return true;
```

Comment:
I always like to set handle to an invalid value (it's not done automatically), because lots of places check `handle.IsInvalid`, no one checks `handle.IsClosed` (except the P/Invoke stub)

```suggestion
            handle = 0;
            return true;
```